crates/editor/src/editor.rs

```diff
@@ -17,7 +17,7 @@ mod blame_entry_tooltip;
 mod blink_manager;
 mod debounced_delay;
 pub mod display_map;
-mod editor_settings;
+pub mod editor_settings;
```

Comment:
I'm guessing there's a better way to do this, but my Rust is very basic. I just wanted to use the `ScrollBeyondLastLine` enum in a test below.

Reply:
We already re-export some members of `editor_settings`, so we can add `ScrollBeyondLastLine` to the list: b2210a11b6